src/python/pants/option/global_options.py

```diff
@@ -46,10 +47,21 @@ class GlobMatchErrorBehavior(Enum):
 class FileNotFoundBehavior(Enum):
     """What to do when globs do not match in BUILD files."""
 
+    ignore = "ignore"
     warn = "warn"
     error = "error"
 
     def to_glob_match_error_behavior(self) -> GlobMatchErrorBehavior:
+        deprecated_conditional(
+            lambda: self == self.ignore,
+            removal_version='1.28.0.dev2',
```

Comment:
I believe this will need to be 1.29.0.dev2 (or 1.29.0.dev0 if you'd like) so that it's deprecated for at least one stable release. See https://www.pantsbuild.org/deprecation_policy.html for the guidance on deprecations, which recently was clarified.

Reply:
Ah, got it! Will do!